### ISSUE TITLE
Focus issue with Design system side navigation in High Contrast Mode

### ISSUE THREAD

**Issue by chrispymm** [ministryofjustice/moj-frontend#1901]
## What

In windows high contrast mode it is hard to see the active/focused menu item because the border is always present.

<img width="436" height="578" alt="Image" src="https://github.com/user-attachments/assets/09433633-3f9a-432c-a233-9bac8747099e" />

There is an issue with forced dark mode on Chrome where the focus states of the side nav and links are a bright yellow with white text. This does not pass contrast standard – I could replicate the issue, but forced dark mode on Chrome is not part of a default testing spec, so would raise as a usability comment.

<img width="579" height="255" alt="Image" src="https://github.com/user-attachments/assets/69d7d58a-d889-455e-a61e-ca7ce865a24a" />

## Why

This is not a WCAG failure, but is a usability issue

## Who needs to be involved

- Developer
- Accessibility Specialist

## Done

Side nav focus is obvious in WHC mode and forced color modes

**Comment by chrispymm** [comment]
@euhyung I've made some updates to the styling now - it should now be acceptable in high contrast modes. It is now possible to distinguish between hover, focus and current page styles. Below are a couple of screenshots, and you can take a a look at a PR here: https://moj-frontend-pr-2332.apps.live.cloud-platform.service.justice.gov.uk/

Let me know if this resolves the issues in this ticket

<img width="1530" height="823" alt="Image" src="https://github.com/user-attachments/assets/0e5fb368-0fe3-452a-a5e4-c8f91c074248" />

Windows 11 High Contrast Mode.  Add another link is focused, API Error is current page, and Button menu is hovered


<img width="1180" height="705" alt="Image" src="https://github.com/user-attachments/assets/cbd121b2-bbd7-4990-9fdd-034997a743ea" />

Mac Chrome, forced-color: active, prefers-color-scheme: dark. Alert is focused, Date picker is current page, and Button menu is hovered

<img width="1030" height="714" alt="Image" src="https://github.com/user-attachments/assets/81988b4c-935b-453c-ba1d-00a823ec94f9" />

API Error is focused, Date picker is current page, and Card is hovered

**Comment by euhyung** [comment]
thanks, focus much more visible. 
(Just as an aside, Chrome forced colours (the initial comment Ben raised) when now set to active, displays on a light theme rather than dark (outside our control?).) 
Can confirm Windows/Edge displays same as what you documented above, in all 4 HCMs. 

<img width="1254" height="665" alt="Image" src="https://github.com/user-attachments/assets/a27b1767-0663-44fb-8650-ecab3e4f5bfb" />

**Comment by chrispymm** [comment]
@euhyung great, thanks for confirming.

For forced colors, there are two emulation settings - `forced-colors: active` and `prefers-color-scheme` you need to set both to "simulate" WHCM

**Comment by euhyung** [comment]
thanks, that's come a long way from the initial screenshots!
